doc/source/whatsnew/v1.0.0.rst

```diff
@@ -249,6 +249,7 @@ Bug fixes
 ~~~~~~~~~
 
 - Bug in :meth:`DataFrame.to_html` when using ``formatters=<list>`` and ``max_cols`` together. (:issue:`25955`)
+- Bug in :meth:`DataFrame.interpolate` where specifying axis by name references variable before it is assigned (:issue:`29132`)
```

Comment:
move this to the Number sub-section. Please *also* move the note right above (the to_html) to the I/O section.